README.md

```diff
@@ -17,13 +17,14 @@ A `Node.js` based command-line client for [tldr](https://github.com/tldr-pages/t
 npm install -g tldr
 ```
 
-If you have trouble with the post-install script, try the following commands:
+If you are installing with sudo, pass the `--user` parameter:
 
 ```bash
-npm install -g --ignore-scripts tldr
-tldr --update
+sudo npm install -g tldr --user=<logged_in_user>
```

Comment:
Yes, color-coded output works. Both in cmd and powershell. Just checked on friend's laptop. But surprisingly `whoami` also works.

But wait, the story doesn't end there. :laughing: Actually, windows doesn't even need the `sudo` part. Simply `npm install -g` works fine without any issues.

So I am guessing its okay to use $(whoami).